src/components/PageWithSideMenu.tsx

```diff
@@ -75,34 +73,33 @@ interface LinkButtonProps extends ButtonProps, Pick<LinkProps, 'to'> {
 }
 
 export const LinkButton = ({ isSelected, ...rest }: LinkButtonProps) => (
-  <li>
-    <Button variant={isSelected ? 'contained' : 'outlined'} size="large" LinkComponent={Link} {...rest} />
-  </li>
+  <Button
+    sx={{ bgcolor: isSelected ? 'primary.main' : 'white', ml: '1rem', width: 'fit-content' }}
```

Comment:
Mulig denne burde ha margin på begge sider? Nå ser det slik ut på siden for redaktør feks 
![bilde](https://user-images.githubusercontent.com/6313468/213407696-a7b4418a-5b52-4490-bb88-b5fcfca9fac4.png)

```suggestion
    sx={{ bgcolor: isSelected ? 'primary.main' : 'white', mx: '1rem', width: 'fit-content' }}
```

Reply:
Men da blir det margin når vi ikke vil ha det også da 😅 feks når vi bruker LinkButtonRow

Reply:
Noe vi kan gjøre er å sette `mx` margin med `> button {` i stedet for `button {`. og så sette margin med `mx` på LinkButtonRow.

Da mener jeg det skal bli margin på knappen bare hvis button er direct child (altså ikke ligger i en div eller noe), så setter vi margin på row i stedet for button når vi skal ha flere

Reply:
Ikke helt skker på hva du mener her🤔

Reply:
ser ut som det er rett nå 👍